src/ngResource/resource.js

```diff
@@ -57,6 +57,24 @@ function shallowClearAndCopy(src, dst) {
  * <div doc-module-components="ngResource"></div>
  *
  * See {@link ngResource.$resource `$resource`} for usage.
+ *
+ * See {@link ngResource.$resourceProvider `$resourceProvider`} for usage.
+ */
+
+/**
+ * @ngdoc provider
+ * @name $resourceProvider
+ *
+ * @description
+ *
+ * Use for configuring {@link ngResource.$resource `$resource`} in module configuration phase.
```

Comment:
I would rather change it to something like:

_Use `$resourceProvider` to change the default behavior of the {@link ngResource.$resource `$resource`} service._

--
<sub>(You don't have to mention the configuration phase. It's not specific to this provider and we don't want to mention such stuff in every provider's documentation.)</sub>